src/github.com/getlantern/lantern/lantern.go

```diff
@@ -79,12 +79,25 @@ func AddLoggingMetadata(key, value string) {
 	logging.SetExtraLogglyInfo(key, value)
 }
 
+//userConfig supplies user data for fetching user-specific configuration.
+type userConfig struct {
+}
+
+func (uc *userConfig) GetToken() string {
+	return ""
+}
+
+func (uc *userConfig) GetUserID() int {
+	return 0
+}
```

Comment:
Great! This is actually almost identical to what I already have so we could just merge it and I'll fix it on my side